wcivf/apps/elections/templates/elections/includes/_people_list_with_lists.html

```diff
@@ -12,7 +12,7 @@
           <h4 class="ds-candidate-name">
             <a href="{{ pp.person.get_absolute_url }}" class="ds-card-link">
               {% if pp.elected %}🌟 <span class="elected_text">Elected:</span> {% endif %}
-              {{ person_post.grouper }}: {{ pp.person.name }}
+              {{ e_post.grouper }}: {{ pp.person.name }}
```

Comment:
looks like a typo? Should it still be `person_post.grouper`?

Reply:
Fixed, thanks for catching.